docs/core/extensions/options.md

```diff
@@ -27,8 +27,8 @@ Create the following `TransientFaultHandlingOptions` class:
 
 An options class:
```

Comment:
@IEvangelist  I think, if you want to reword the list below like you did, you also need to reword this paragraph.